src/fetch.js

```diff
@@ -1,17 +1,39 @@
 import addVersion from './api/addVersion'
+import isSameOrigin from './utils/isSameOrigin'
+
+/**
+ * Returns the parsed URL for the specified request
+ * @param {Request|String} request A request instance or a URL string
+ * @return {URL}
+ */
+function getURL(request) {
+  let url = request.url
+
+  if (typeof request === 'string') {
+    url = request
+  }
+
+  return new URL(url, window.location.href)
+}
 
 // Here we patch fetch and XMLHttpRequest to always add version parameter to api calls so that cached results
 // from previous versions of the app aren't served to new versions.
 /* istanbul ignore else */
 if (typeof window !== 'undefined') {
   const originalFetch = window.fetch
 
-  window.fetch = function rsfVersionedFetch(url, init) {
-    if (url.url) {
-      // the first param can be a request object
-      url = new Request(addVersion(url.url).toString(), url)
+  window.fetch = function rsfVersionedFetch(url, init, ...others) {
```

Comment:
Did you forget to remove the `...others`?

Reply:
Fixed